src/containers/sitePage/index.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import SelectorMap from '../../components/mapComponents/maps/selectorMap';
+import { MapGeoDataReducerState } from '../../components/mapComponents/ducks/types';
+import { asyncRequestIsComplete } from '../../utils/asyncRequest';
+import PageLayout from '../../components/pageLayout';
+import { Col, Form, message, Row, Typography } from 'antd';
+import styled from 'styled-components';
+import { DARK_GREEN } from '../../utils/colors';
+import PageHeader from '../../components/pageHeader';
+import { useParams } from 'react-router-dom';
+import { SiteProps } from '../treePage/ducks/types';
+import Client from '../../api/apiClient';
+import ProtectedClient from '../../api/protectedApiClient';
+import { C4CState } from '../../store';
+import { connect } from 'react-redux';
+import SiteFeatures from '../../components/siteFeatures';
+import SiteEntryTable from '../../components/siteEntryTable';
+import UpdateSiteForm from '../../components/forms/updateSiteForm';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/windowDimensions';
+import {
+  EditSiteRequest,
+  UpdateSiteRequest,
+} from '../../components/forms/ducks/types';
+import SelectorMapDisplay from '../../components/mapComponents/mapDisplays/selectorMapDisplay';
+
+const { Text } = Typography;
+
+const SitePageContainer = styled.div`
+  width: 90%;
+  margin: 8vh auto;
+`;
+
+const SectionHeader = styled(Text)`
+  font-weight: bold;
+  font-size: 20px;
+  color: ${DARK_GREEN};
+`;
+
+const MobileMapContainer = styled.div`
+  margin: 15px 0;
+  height: 300px;
+`;
+
+const MarginBottomRow = styled(Row)`
+  margin-bottom: 30px;
+`;
+
+interface SiteParams {
+  id: string;
+}
+
+interface SitePageProps {
+  readonly neighborhoods: MapGeoDataReducerState['neighborhoodGeoData'];
+  readonly sites: MapGeoDataReducerState['siteGeoData'];
+}
+
+const SitePage: React.FC<SitePageProps> = ({ neighborhoods, sites }) => {
+  const [site, setSite] = useState<SiteProps>();
+  const id = Number(useParams<SiteParams>().id);
+  const { windowType } = useWindowDimensions();
+
+  const [editSiteForm] = Form.useForm();
+  const [updateSiteForm] = Form.useForm();
+
+  const getSite = () => {
+    // TODO: BUG - getSite isn't getting neighborhood id
```

Comment:
Did some further investigation and it turns out the backend returns the neighborhood name instead of ID. We can either update the docs + this frontend code or the backend. I'd prefer the backend, but open to any reasons for keeping the backend as is. Currently, this discrepancy just means the edit site page will always say 'No neighborhood found.' instead of the neighborhood name, which isn't ideal but doesn't break anything

Reply:
This is a quick change